sdcm/sct_events/operator.py

```diff
@@ -28,74 +30,41 @@ class ScyllaOperatorLogEvent(LogEvent):
-        return super().msgfmt + " {0.trace_id} {0.namespace}" + cluster + ": {0.message}, {0.error}"
+    source: str
 
-    # pylint: disable=too-many-arguments
-    def set_params(self, timestamp=None, namespace=None, cluster=None, message=None, error=None, trace_id=None):
-        self.namespace = namespace
-        self.cluster = cluster
+    def __init__(self, message: str = '', source: str = '', regex: str = None, severity=Severity.NORMAL):
+        super().__init__(regex=regex, severity=severity)
         self.message = message
-        self.timestamp = timestamp
-        self.error = error
-        self.trace_id = trace_id
-
-    def load_from_json_string(self: T_log_event, data: str) -> Optional[T_log_event]:
-        try:
-            log_data = json.loads(data)
-        except Exception as exc:  # pylint: disable=broad-except
-            LOGGER.error('Failed to interpret following log line:\n%s\n, due to the: %s', data, exc)
-            return None
-        event_data = {}
-        for log_en_name, attr_name in self.event_data_mapping.items():
-            log_en_data = log_data.get(log_en_name, None)
-            if log_en_data is None:
-                continue
-            if attr_name == 'timestamp':
-                log_en_data = datetime.strptime(log_en_data, "%Y-%m-%dT%H:%M:%S.%fZ").timestamp()
-            event_data[attr_name] = log_en_data
+        self.source = source
+        self.line_number = None
+
+    def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_event:
+        # I0628 15:53:02.269804       1 operator/operator.go:133] <message>
+        splits = line.split(maxsplit=4)
+        if len(splits) != 5 or len(splits[0]) != 5:
+            return
+        type_month_date, time_string, _, self.source, self.message = splits
         try:
-            self.set_params(**event_data)
+            self.timestamp = dateutil.parser.parse(
+                str(datetime.datetime.now().year) + '-' + type_month_date[1:3] + '-' +
+                type_month_date[3:5] + ' ' + time_string.split('.')[0]).timestamp()
         except Exception as exc:  # pylint: disable=broad-except
-            LOGGER.error('Failed to set event parameters %s\n, due to the: %s', data, exc)
-            return None
-        return self
+            self.timestamp = time.time()
 
+    @property
+    def msgfmt(self):
+        return super().msgfmt + ": {0.source} {0.message}"
 
 ScyllaOperatorLogEvent.add_subevent_type(
     "REAPPLY", severity=Severity.WARNING,
     regex="please apply your changes to the latest version and try again")
 ScyllaOperatorLogEvent.add_subevent_type(
     "TLS_HANDSHAKE_ERROR", severity=Severity.WARNING,
-    regex="TLS handshake error from .*: EOF")
+    regex="TLS handshake error from .*")
 ScyllaOperatorLogEvent.add_subevent_type(
     "OPERATOR_STARTED_INFO", severity=Severity.NORMAL,
-    regex="Starting the operator...")
+    regex="] operator version ")
```

Comment:
Here is how looks like line where operator really starts:

```
...
I0816 08:45:02.840256       1 scyllacluster/controller.go:205] "Starting controller" controller="ScyllaCluster"
...
```

Reply:
fixed